stage1/units/units/ctrl-alt-del.target

```diff
@@ -0,0 +1,6 @@
+[Unit]
+Description=Reboot
+DefaultDependencies=no
+Requires=systemd-reboot.service
+After=systemd-reboot.service
+AllowIsolate=yes
```

Comment:
On my system, and according to systemd.special(7), this should be a symlink:
```
ctrl-alt-del.target -> reboot.target
```
But I don't know if it is actually needed.